awscli/examples/rum/list-rum-metrics-destinations.rst

```diff
@@ -0,0 +1,25 @@
+**To return a list of destinations that you have created to receive RUM extended metrics, for the specified app monitor**
+
+The following ``list-rum-metrics-destinations`` returns a list of destinations that you have created to receive RUM extended metrics, for the specified app monitor. ::
+
+	aws rum list-rum-metrics-destinations \
```

Comment:
Same here. All of these indents need to be 4 spaces each and not tab indenting.
For more information: https://aws.github.io/aws-cli/docs_styleguide.html#general